functional_testing/Grid_API_Testing/api_testing/testcases/basic_tests/test_storageclusters_apis.py

```diff
@@ -0,0 +1,111 @@
+from random import randint
+from api_testing.testcases.testcases_base import TestcasesBase
+from api_testing.grid_apis.apis.storageclusters_apis import Storageclusters
+import unittest
+
+class TestStorageclustersAPI(TestcasesBase):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.storageclusters_api = Storageclusters()
+
+    def setUp(self):
+        super(TestStorageclustersAPI, self).setUp()
+        self.lg.info('Deploy new storage cluster (SC0)')
+        self.label = self.rand_str()
+        self.servers = randint(1,3)
+        self.types = ['nvme', 'ssd', 'hdd', 'archive']
+        self.drivetype = self.random_item(self.types)
+        self.slaveNodes = self.random_item([True, False])
+        self.nodes = [self.get_random_node()]
+        self.body = {"label": self.label,
+                     "servers": self.servers,
+                     "driveType": self.drivetype,
+                     "slaveNodes": self.slaveNodes,
+                     "nodes":self.nodes}
+        self.storageclusters_api.post_storageclusters(self.body)
+
+    def tearDown(self):
+        self.lg.info('Kill storage cluster (SC0)')
+        self.storageclusters_api.delete_storageclusters_label(self.label)
+        super(TestStorageclustersAPI, self).tearDown()
+
+    def test001_get_storageclusters_label(self):
+        """ GAT-001
+        **Test Scenario:**
+        #. Deploy new storage cluster (SC0)
+        #. Get storage cluster (SC0), should succeed with 200
+        #. Get nonexisting storage cluster (SC0), should fail with 404
+        """
+        self.lg.info('Get storage cluster (SC0), should succeed with 200')
+        response = self.storageclusters_api.get_storageclusters_label(self.label)
+        self.assertEqual(response.status_code, 200)
+        for key in ['label', 'driveType', 'nodes']:
+            self.assertEqual(response.json()[key], self.body[key])
+
```

Comment:
test status of created storagecluster , should be empty, deploying, ready or error ?